server/main.lua

```diff
@@ -193,3 +193,14 @@ QBCore.Functions.CreateCallback("qb-multicharacter:server:getSkin", function(_,
         cb(nil)
     end
 end)
+
+QBCore.Commands.Add("deletechar", "Deletes another players character", {{name = "Citizen ID", help = "The Citizen ID of the character you want to delete"}}, false, function(source,args)
+    local Player = QBCore.Functions.GetPlayer(source)
+
+    if args and args[1] then
+        QBCore.Player.ForceDeleteCharacter(tostring(args[1]))
+        TriggerClientEvent("QBCore:Notify", Player.PlayerData.source, "You successfully deleted the character with citizen id \"" .. tostring(args[1]) .. "\".")
+    else
+        TriggerClientEvent("QBCore:Notify", Player.PlayerData.source, "You forgot to input a citizen id", "error")
+    end
```

Comment:
```suggestion
    if args and args[1] then
        QBCore.Player.ForceDeleteCharacter(tostring(args[1]))
        TriggerClientEvent("QBCore:Notify", source, "You successfully deleted the character with citizen id \"" .. tostring(args[1]) .. "\".")
    else
        TriggerClientEvent("QBCore:Notify", source, "You forgot to input a citizen id", "error")
    end
```
I think the `Player` variable can be removed here since it's only used for the player ID anyway.